### ISSUE TITLE
Start Position bug

### ISSUE THREAD

**Issue by BlazONBZ** [TheSillyDoggo/GeodeMenu#77]
As of 1.5.9 when using Start Pos trigger and playing the level (not in editor) it somehow becomes a spider. But when playtesting in editor, Start pos works as intended.

**Comment by JerichoGesolgani** [comment]
yeah this started to happening to me ig i have to download a older version and you should to what i hate is that dev is responsing the fucking issue like bro fix your fix shit just pleases

**Comment by Lilicogamer13** [comment]
> yeah this started to happening to me ig i have to download a older version and you should to what i hate is that dev is not responsing the fucking issue like bro fix your shit just pleases

bro calm down this is github not twitter, stop spamming bad words

btw you can fix it by reseting the settings of every start pos(on the level editor)
